packages/ember/lib/index.js

```diff
@@ -127,7 +127,6 @@ Ember.observersFor = metal.observersFor;
 Ember.removeObserver = metal.removeObserver;
 Ember._suspendObserver = metal._suspendObserver;
 Ember._suspendObservers = metal._suspendObservers;
-Ember.required = metal.required;
```

Comment:
I believe we are going to have to guard this

Reply:
ok